ZON-2409: Import new fields from Brightcove

sallner opened the pull request:
Needs https://github.com/ZeitOnline/zeit.content.video/pull/4

sallner committed:
re ZON-2409: Implement `credit` attribute on `Video`.

sallner committed:
re ZON-2409: Read authors from brightcove.

sallner committed:
re ZON-2409: Rename `credit` to `video_still_copyright`.

wosc commented on setup.py:
```diff
@@ -29,6 +29,9 @@
         'zope.interface',
         'zope.schema',
     ],
+    extras_require=dict(test=[
```
Wir benutzen im vivi bisher keine Test-Extras... ich fände das zwar auch sinnvoll, aber Zagy war irgendwie dagegen. Und wenn dann sollte man es vermutlich einmal überall durchziehen (sonst wird der Buildout das völlige Chaos), insofern würd ich es akut vermutlich eher nicht tun. ;)

Was mir letztens übrigens schon ein paarmal aufgefallen ist, dass zeit.solr zu Kreisreferenzen neigt, weil einerseits zeit.solr von div. Content-Typen abhängt, um sie zu indizieren, aber andererseits div. Content-Typen von zeit.solr abhängen, weil sie für UI-Tests etc. die Suche brauchen.

sallner replied:
Wir haben in `zeit.addcentral` und `zeit.wysiwyg` bereits extras_require gefunden und es daher übernommen. Sollten wir es dann dort auch entfernen? Oder lassen wir es jetzt stehen um mal den Anfang zu machen?

wosc replied:
Ah. Na dann können wir meinetwegen grassroots-mäßig langsam Test-Extras einführen. :)

wosc commented on src/zeit/brightcove/converter.py:
```diff
@@ -308,6 +309,16 @@ def renditions(self):
+                   for author in data_authors.split(' ')]
+
+        video = zeit.content.video.video.Video()
+        return tuple([video.authorships.create(author)
```
Man _﻿⁠⁠könnte﻿⁠⁠_ jetzt argumentieren, bei Videos kommt es nicht vor, dass Referenzen ein Eigenleben haben, und dort die authorships-Property mit nem angepassten `﻿⁠⁠⁠⁠MultiResource﻿⁠⁠⁠⁠` überschreiben, das das `create` beim `__set__` übernimmt. Bin allerdings nicht sicher, ob das dadurch global gesehen so viel besser würde (insb was die formlib dann macht)... Was meint ihr?

sallner committed:
re ZON-2409: No longer create temporary object for transfer of `authorships`.

 - We use the new convenience from `zeit.cms` now, to pass `ICMSContent`
   directly to `ReferenceProperty` without creating a `IReference` first.

sallner committed:
re ZON-2409: Update version dependencies.

wosc commented:
Find ich prima, wartet noch auf https://github.com/ZeitOnline/zeit.content.video/pull/4

wosc commented:
![author at work](http://i.giphy.com/XIqCQx02E1U9W.gif)

wosc committed:
Update dependencies
